scripts/run-lh.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+LIQUID="$HOME/liquidhaskell/.cabal-sandbox/bin/liquid"
+$LIQUID --version
+echo "-------------------------------------------------"
+pwd
+echo "-------------------------------------------------"
+cd ./raaz-core
+pwd
+echo "-------------------------------------------------"
+find . -name '*.hs' -print0 | xargs -n 1 -0 -I{} sh -c "echo ---- {}; $LIQUID {}"
+cd ..
+pwd
+echo "-------------------------------------------------"
+cd raaz-implementation
+pwd
+echo "-------------------------------------------------"
+find . -name '*.hs' -print0 | xargs -n 1 -0 -I{} sh -c "echo ---- {}; $LIQUID {}"
```

Comment:
`xargs` or the find version I guess runs liquid haskell checks one at a time on each file. I am not sure if it is the correct thing to do. Shouldn't liquidhaskell get all the relevant files as command like arguments at one go as some of the files have dependencies ?

On in other words is something morally equivalent to
 
```bash
liquidhaskell `find . -name '*.hs'`
``` 
better ?

Reply:
One issue with this is that LH complains when the same module name is found in multiple files. In this case, the Raaz.Entropy module.

As far as the dependencies are concerned, LH is able to identify them as long as there's a cabal file in the directory.

Reply:
The Entropy module should be ignored. I do not think there is anything there that is of interest to LH